libs/cardano-ledger-core/src/Cardano/Ledger/Hashes.hs

```diff
@@ -68,7 +68,7 @@ data EraIndependentScriptIntegrity
 
 type EraIndependentWitnessPPData = EraIndependentScriptIntegrity
 
--- $scriptHash
+-- scriptHash
```

Comment:
This was meant to be documentation for the header file. See `$scriptHash` anchor in the exports section. I think it would make sense to delete it from there too. A better alternative would be to revert this change and some haddock here.

Reply:
I went to add haddock, but there's really nothing to say, so I've just removed it.